controllers/listActions/list.actions.js

```diff
@@ -0,0 +1,88 @@
+const List = require('../../models/list.model');
+const Item = require('../../models/item.model');
+const { resolve500Error } = require('../../middlewares/validation');
+const { checkIsSomethingDeletedByIds } = require('../../utils/utils');
```

Comment:
"checkIsSomethingDeletedByIds" - I think plural in "ids" not very logical, because usually you delete something by id (just one)